src/components/MapAddLayer.vue

```diff
@@ -81,6 +81,11 @@ import { Chrome as ChromeComponent } from 'vue-color';
 
 import { EPSG } from 'app/constant';
 
+/**
+ * @since v3.8
+ */
+const Projections = require('g3w-ol/projection/projections');
```

Comment:
The `@since` tag should only be used where you export something that is also accessible from outside that file (functions, variables, properties, classes, ...). If so, please double check and place the comment in a above the "exported" item, if not, just remove it